### DISCUSSION TITLE
how can i get galaxy brain?

### DISCUSSION THREAD

**Discussion by TahaBakhtari** [drknzz/GitHub-Achievements#354] [General]
can you help with that?

**Comment by TahaBakhtari** [comment]
kj

**Comment by gitagogaming** [comment]
You can get the `Galaxy Brain` achievement by answering various discussions and having your answer accepted. 

The first achievement is at 2 answers accepted, then bronze is 8, silver is 16 and gold is 32

**Reply by jvcByte** [reply]
How can i find repo to participate in discussions?

**Reply by ghost** [reply]
Discussion authors and users with the triage role or greater for a repository can mark a comment as the answer to a discussion in the repository. Similarly, discussion authors and users with the triage role or greater for the source repository for organization discussions can mark a comment as the answer to a discussion in the organization.

You can mark a comment in the discussion as an answer to the discussion if a discussion is within a category that accepts answers
